src/utilities/filterUtilities.js

```diff
@@ -0,0 +1,42 @@
+const applySortingChoice = (givenState, providedProducts) => {
+    if (givenState.sortPriceOrder === "Low To High") {
+        return [...providedProducts].sort((a, b) => Number(a.discountedPrice) - Number(b.discountedPrice))
+    } else if (givenState.sortPriceOrder === "High To Low") {
+        return [...providedProducts].sort((a, b) => Number(b.discountedPrice) - Number(a.discountedPrice))
+    } else {
+        return [...providedProducts]
```

Comment:
suggestion: While returning the exact same arr you don't need to create a copy first.